src/Pages/News.js

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import { Spinner } from "reactstrap";
+import NewsList from "../Components/News/NewsList";
+import NoResults from "../Components/Shared/NoResults";
+import Pagination from "../Components/Shared/Pagination";
+import DotCMSApi from "../libs/dotcms.api";
+import "./News.css";
+
+class NewsPage extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+    this.state = {
+      loading: true,
+      news: [],
+      query: `{ 
+            "query": {
+                "bool": {
+                    "must": {
+                        "query_string" : {
+                            "query" : "+contentType:news +languageId:1 +conHost:CONFHOSTVALUE"
+                        }
+                    }
+                }
+            },
+            "sort" : [
+                { "SORTBYVALUE" : {"order" : "SORTTYPEVALUE"}}
+            ],
+            "from": OFFSETVALUE,
+            "size": SIZEPERPAGE
+        }}`,
+      url: "/api/es/search"
+    };
+
+    this.pageChange = this.pageChange.bind(this);
+  }
+
+  async getNewsData(query) {
+    return DotCMSApi.request({
+      url: this.state.url,
+      method: "POST",
+      body: query
+    }).then(response => response.json());
+  }
+
+  fillQuery(fetchParams) {
```

Comment:
All this logic shouldn't be in a component, all this is query logic, we should encapsulate all this in the lib.